sdk/RELEASE.md

```diff
@@ -12,6 +12,21 @@
 
 ## Bug Fixes and Other Changes
 
+## Documentation Updates
+# Current Version (2.0.0-beta.0)
+
+## Major Features and Improvements
+
+## Breaking Changes
+
+### For Pipeline Authors
+
+### For Component Authors
+
+## Deprecations
+
+## Bug Fixes and Other Changes
```

Comment:
Don't we have some bug fixes since last alpha?
Either way, feel free to merge this by yourself.